.github/ISSUE_TEMPLATE/checkpoint.md

```diff
@@ -2,7 +2,7 @@
 name: CP Review
 about: Checkpoint Review checklist
 title: ''
-labels: CP Review, Pri1, Sharing
+labels: CP Review, Sharing
```

Comment:
In the typescript PR, the Sharing label is removed and Pri1 is kept - not sure which deletion (if any) were intended.
(Have a similar comment in typescript PR)